docs/production.md

```diff
@@ -142,12 +330,31 @@ and referenced from the `site` and `production` environments, e.g.:
   Note that any time spent blocked due to this parallelism limit does not count
   against the (un-overridable) internal OpenTofu timeout of 30 minutes
 
-- By default, OpenStack Nova also [limits](https://docs.openstack.org/nova/latest/configuration/config.html#DEFAULT.max_concurrent_builds)
+- By default, OpenStack Nova also
+  [limits](https://docs.openstack.org/nova/latest/configuration/config.html#DEFAULT.max_concurrent_builds)
   the number of concurrent instance builds to 10. This is per Nova controller,
-  so 10x virtual machines per hypervisor. For baremetal nodes it is 10 per cloud
-  if the OpenStack version is earlier than Caracel, else this limit can be
-  raised using [shards](https://specs.openstack.org/openstack/nova-specs/specs/2024.1/implemented/ironic-shards.html).
+  so 10x virtual machines per hypervisor. For baremetal nodes it is 10 per
+  cloud if the OpenStack version is earlier than Caracel, else this limit can
+  be raised using
+  [shards](https://specs.openstack.org/openstack/nova-specs/specs/2024.1/implemented/ironic-shards.html).
   In general it should be possible to raise this value to 50-100 if the cloud
   is properly tuned, again, demonstrated through testing.
 
-- Enable alertmanager if Slack is available - see [docs/alerting.md](./alerting.md).
+- Enable alertmanager if Slack is available - see
+  [docs/alerting.md](./alerting.md).
+
+- By default, the appliance uses a built-in NFS share backed by an OpenStack
+  volume for the cluster home directories. You may find that you want to change
+  this. The following alternatives are supported:
+
+  - External NFS <!--- External NFS docs TODO ---> 
+  - CephFS via OpenStack Manila <!--- filesystems docs TODO ---> 
+  - [Lustre](../roles/lustre/README.md)
```

Comment:
Open question: are the any others I've missed?

Reply:
I think you can leave out external NFS docs for the moment really, we can add those into filesystems at some point. Generally I think using NFS is a bad idea!